commit/merkleroot/rmn/config.go

```diff
@@ -0,0 +1,37 @@
+package rmn
+
+import (
+	"crypto/ed25519"
+
+	mapset "github.com/deckarep/golang-set/v2"
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+)
+
+// Config contains the RMN configuration required by the plugin and the RMN client in a single struct.
+type Config struct {
+	Home   RMNHomeConfig
+	Remote RMNRemoteConfig
+}
+
+type RMNHomeConfig struct {
+	RmnNodes         []RMNNodeInfo
+	ConfigDigest     cciptypes.Bytes32
+	RmnReportVersion string // e.g. "RMN_V1_6_ANY2EVM_REPORT"
```

Comment:
Is this actually fetchable from the contract?

Reply:
not sure yet, haven't looked much into configs, I just assume I have it.. somehow.
cc @0xnogo

Reply:
I'm not sure yet. The only place where it could be in the RMNHome is in the `bytes offchainConfig`. But we need to clarify what's going to go there.

Reply:
It's a constant in the contract

Reply:
https://github.com/smartcontractkit/chainlink/blob/c83c68735bdee6bbd8510733b7415797cd08ecbd/contracts/src/v0.8/ccip/rmn/RMNRemote.sol#L9